FirebaseRemoteConfig/Sources/RCNFetch.m

```diff
@@ -249,7 +252,8 @@ - (void)refreshInstanceIDTokenAndFetchCheckInInfoWithCompletionHandler:
               [NSString stringWithFormat:@"Error getting iid : %@.", error];
           FIRLogError(kFIRLoggerRemoteConfig, @"I-RCN000055", @"%@",
                       [NSString stringWithFormat:@"%@", errorDescription]);
-          return [self
+          strongSelfQueue->_settings.isFetchInProgress = NO;
```

Comment:
Are we sure `isFetchInProgress` is accessed in thread safe way?   Sorry, it's hard to see from the PR itself...

Reply:
Ah, ok, I can see that it is an `atomic` property now, so should be good.